draft-ietf-rtcweb-jsep.xml

```diff
@@ -1217,8 +1210,8 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host ]]></sourcecode>
           contains an SDP answer per <xref target="RFC3264" format="default"/> with the supported
           configuration for the session that is compatible with the
           parameters supplied in the most recent call to
-          setRemoteDescription, which <bcp14>MUST</bcp14> have been called prior to
-          calling createAnswer. Like createOffer, the returned blob
+          setRemoteDescription (setRemoteDescription <bcp14>MUST</bcp14> have been called prior to
```

Comment:
What's the intent here?

Reply:
I just felt like it was run-on. Don't feel strongly about this.

Reply:
I see what you're getting at, but I'm not crazy about the parenthetical. I'm inclined to leave it as-is here.

Reply:
I am also not crazy about the parenthetical. If we can't improve it, I'm OK with it as-is.

Reply:
PTAL at my suggestion of a semicolon

Reply:
LGTM.